Spanner/src/V1/Gapic/SpannerGapicClient.php

```diff
@@ -308,6 +308,14 @@ public static function parseName($formattedName, $template = null)
      */
     public function __construct(array $options = [])
     {
+        if (isset($options['enable-gcp'])) {
+            $api = 'spanner';
+            $conf_path = "./../../$api.grpc.config";
```

Comment:
line 314 to 317 can be wrapped by a function and put into the `GapicClientTrait`. But we still at least set the `conf_path` here.